src/LCT.Common/PublishPipelineArtifacts.cs

```diff
@@ -0,0 +1,58 @@
+﻿using LCT.Common.Runtime;
+using log4net;
+using log4net.Core;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace LCT.Common
+{
+    public static class PublishPipelineArtifacts
```

Comment:
Do you think the name PipelineArtifactUploader would be better for the class
```suggestion
    public static class PublishPipelineArtifacts
```